### PR TITLE
fix: share layout between /sites/** and /settings

### PR DESCRIPTION
**Pull request by dymbe**
Move some of the shared elements from /sites* and /settings into a shared layout to avoid repetition and speed up rendering slightly. To allow /settings and /sites* to share layout, I moved both into a route group called (main).

Closes https://github.com/openearthplatforminitiative/issues/issues/134

### CODE REVIEW

File: frontend/app/[lang]/(main)/sites/page.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                intensity={properties.intensity}
+                timing={properties.peak_timing}
+                siteName={user.sites[index].name}
+              />
+            );
+          })
+        )}
+
+        <List>
+          <List>
```
Comment (daniejoh) [inline frontend/app/[lang]/(main)/sites/page.tsx]: Double list on purpose? If so why?

File: frontend/app/components/Navbar.tsx [hunk 2]
```diff
@@ -31,8 +34,7 @@ const Navbar = ({ dict, lang, currentPath }: NavbarProps) => {
       >
         <Box
           sx={{
-            backgroundColor:
-              currentPath === `/${lang}/sites` ? '#D5E7D6' : undefined,
+            backgroundColor: segment === 'sites' ? '#D5E7D6' : undefined,
```
Comment (daniejoh) [inline frontend/app/components/Navbar.tsx:37]: Why undefined, if so does it get color from a theme?

**Review by daniejoh — CHANGES_REQUESTED** [review]

File: frontend/app/components/Navbar.tsx (hunk 2 above)
Comment (daniejoh) [reply frontend/app/components/Navbar.tsx:37]: I see it used to be like that, just curious 😄

File: frontend/app/[lang]/(main)/sites/page.tsx (hunk 1 above)
Comment (dymbe) [reply frontend/app/[lang]/(main)/sites/page.tsx]: Definitely not on purpose, nicely spotted! :)

File: frontend/app/components/Navbar.tsx (hunk 2 above)
Comment (dymbe) [reply frontend/app/components/Navbar.tsx:37]: Setting it to undefined should be equivalent to not setting it at all. Only the link to the current sub-page should be highlighted with a background-color

**Pull request merged**